pkg/scheduler/plugins/reservation/plugin.go

```diff
@@ -634,6 +696,12 @@ func (pl *Plugin) makePostFilterReasons(state *stateData, filteredNodeStatusMap
 	// to prepare the exact messages by ourselves.
 	var reasons []string
 	var b strings.Builder
+	if nameUnmatched > 0 {
+		b.WriteString(strconv.Itoa(nameUnmatched))
+		b.WriteString(" Reservation(s) didn't match the requested reservation name")
```

Comment:
will this message get cut due to out of length

Reply:
yes, however the msg probably not longer than the max length 1024